docs/configuration.md

```diff
@@ -214,6 +222,11 @@ Apart from these, the following properties are also available, and may be useful
   <td>(none)</td>
   <td>
     A string of extra JVM options to pass to the driver. For instance, GC settings or other logging.
+    
+	<br /><em>Note:</em> In client mode, this config must not be set through the <code>SparkConf</code>
+	directly in your application, because the driver JVM has already started at that point.
+	Instead, please set this through the command line option (see Dynamically Loading Spark Properties) 
+	or in your default properties file.</td>
```

Comment:
instead of `(see Dynamic Loading Spark Properties)`, we should be more specific here with `--driver-java-options`